### PR TITLE
Test part two

### PR DESCRIPTION
**Pull request by Kidd254**
in this pull request me and my partner did the following: 

- Created a test file)for a file 
- wrote tests for the clear, completed and add to list functions
- used mock storage object to "imitate" localStorage operations

### CODE REVIEW

File: src/modules/test/partTwo.test.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  it('creates an input element with the specified properties', () => {
+    const description = 'Finish project report';
+    const result = createTaskDescription(description);
+    expect(result).toBeInstanceOf(HTMLInputElement);
+    expect(result.getAttribute('type')).toBe('text');
+    expect(result.getAttribute('name')).toBe('edit-todo');
+  });
+});
+
+describe('checkboxStatus', () => {
```
Comment (victorokoroji) [inline src/modules/test/partTwo.test.js]: - [ ] One of your test seems to be failing. Kindly fix it 👍🏽 

image

File: src/modules/test/partTwo.test.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
+/* eslint-disable */
```
Comment (victorokoroji) [inline src/modules/test/partTwo.test.js]: - [ ] Kindly ensure you dont disable any of the linters as linters helps to reduce bugs in your codebase and also focus you to write workable code. Please remove the linters you disabled

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @Kidd254, 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the Github Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 

## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OyePriscilla — APPROVED** [review]
Hi @Kidd254 and @Eldonratzlaff 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Github Flow observed✔️
Linters Set up correctly ✔️ 
Descriptive PR summary and title ✔️ 
All test passed ✔️ 

## Optional suggestions

- [optional] Please kindly add a working link to the live demo section of your README file (for both online version and presentation), if it's not available at the moment; kindly indicate **`NOT AVAILABLE AT THE MOMENT`**.. 👍  Thanks. 🙏 

- [optional] I observed that your third partner did not contribute to this milestone, kindly take note that your PR will be invalidated subsequently if perhaps such happen again. The essence of the collaboration is to prepare us ahead for our future job. 👍 

- [optional] Commit messages suppose to be in imperative tense and capitalized, please take note of this for subsequent PRs. Thanks. ✋   

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**